### ISSUE TITLE
Repl not launched in debug mode despite the setting being on

### ISSUE THREAD

**Issue by cgrand** [laurentpetit/ccw#849]
I (as many people) have something like `:jvm-opts ^:replace []` in their `profiles.clj` and because of that the `update-in` on `:jvm-opts` is ignored and the REPL is not started in debug mode.

**Comment by laurentpetit** [comment]
Indeed, it's totally reproducible on the Leiningen (2.5.2 at least) command line, without involving CCW.

**Comment by laurentpetit** [comment]
For the record, I have created this Leiningen Issue to help progress on this CCW issue https://github.com/technomancy/leiningen/issues/1969

**Comment by laurentpetit** [comment]
Phil just acknowledged this is a bug in leiningen.
